overcast/runner/__init__.py

```diff
@@ -202,18 +215,17 @@ def build(self):
 
         nics = [{'port-id': port_id} for port_id in self.create_nics(self.info['networks'])]
 
-        volume = self.runner.get_cinder_client().volumes.create(size=self.info['disk'],
-                                                                imageRef=self.info['image'])
-        self.record_resource('volume', volume.id)
-
-        while volume.status != 'available':
-            time.sleep(3)
-            volume = self.runner.get_cinder_client().volumes.get(volume.id)
-
-        bdm = {'vda': '%s:::1' % (volume.id,)}
+        if self.info['boot_volume']:
+            bdm_v2 = [{'source_type': 'image', 'uuid': self.info['image'],
+                        'destination_type': 'volume', 'volume_size': self.info['boot_volume'],
+                        'delete_on_termination': 'true', 'boot_index': '0'}]
+            image = None
+        else:
+            image = self.info['image']
+            bdm_v2 = None
```

Comment:
Hm, let me rephrase: *I* could not get bdmv2 to work on HP Cloud. If HP Cloud now supports bdmv2 and/or you've found a way to make it work, that's fine, but it needs to work there, too.